test/unit/driver-specs.js

```diff
@@ -171,12 +171,14 @@ describe('driver', () => {
       driver.opts.udid = "01234567889";
       driver.isEmulator().should.equal(false);
     });
-    it('fingerprint should be rejected if avd is not defined', () => {
-      driver.opts = {};
+    it('fingerprint should be rejected if isEmulator is false', () => {
+      driver.opts.avd = undefined;
```

Comment:
Alternatively, you can mock the `isEmulator` call to return false, and assert that it is called.